### DISCUSSION TITLE
BT-198(BT-539)-Lot Unpublished Access Date

### DISCUSSION THREAD

**Discussion by jaggaer-fgeniola** [OP-TED/eForms-SDK#784] [Q&A] [answered]
Good afternoon,
If I understand correctly, this field can be left empty if you do not want to make sensitive data public but then the validation fails indicating that a date between 2 days and 3 years of the publication date must be entered.
In the xml, if a value is not set, the element is not created.

Where am I wrong?
Thank you.

**Comment by YvesJo** [comment, marked as answer]
Hi,
Allowing the field to be empty has been prepared with SDK 1.9 and is effective since SDK 1.10.

**Comment by jaggaer-fgeniola** [comment]
Thanks for the reply, we tried to validate a notice via the "enotices2" preview site and no error message is shown, how is it possible that the CVS validation service returns an error to us and on the "enotices2" site no?
Thanks

**Reply by YvesJo** [reply]
I know there are some restrictions about the use of enotices2 preview and invite you to look at https://docs.ted.europa.eu/home/eforms/preview/#_enotices2 and/or contact the helpdesk regarding this.
Also be aware that enotices2 is always using the latest version (currently SDK 1.10)
